src/services/env/Env.ts

```diff
@@ -27,9 +27,10 @@ export default class Env {
     const config = readPathConfigFromDom()
     const version = semver(env('KUMA_VERSION', config.version))
 
+    const productName = env('KUMA_PRODUCT_NAME').split(' ').join('-')
```

Comment:
isn't replace a better way to do this?
```suggestion
    const productName = env('KUMA_PRODUCT_NAME').replace(' ', '-')
```
Also should we should uriEncode instead? (We don't really use this utm atm so breaking change should be fine)

Reply:
Also if you clean this remove the (GUI) bit in the utm. GUI is the most common place no need to be specific IMO

Reply:
I agree; if we use `encodeURIComponent`, this would allow a consuming side to decode it reliably whereas with dashes, you don’t know what should be a dash and what should be a space.

```suggestion
    const productName = encodeURIComponent(env('KUMA_PRODUCT_NAME'))
```

If you go with the replace call, it should use `replaceAll` to replace all instances not just the first.

```suggestion
    const productName = env('KUMA_PRODUCT_NAME').replaceAll(' ', '-')
```

Reply:
> isn't replace a better way to do this?

Yes probably, but watch for one of my most common mistakes - not using `replaceAll` :highfive: 😄 . Much prefer to uri encode it so I'll do that instead, and remove the `GUI` bit, gimme 2 tics. Oh! 🤔 do we even need the `utm_medium=` bit?